src/Microsoft.ServiceFabric.Services/Runtime/StatefulServiceReplicaAdapter.cs

```diff
@@ -195,11 +195,31 @@ async Task IStatefulServiceReplica.CloseAsync(CancellationToken cancellationToke
 
             await this.CloseCommunicationListenersAsync(cancellationToken);
             await this.CancelRunAsync();
+            ServiceTrace.Source.WriteInfoWithId(
+                TraceType,
+                this.traceId,
+                "Calling userServiceReplica.OnCloseAsync()");
+
             await this.userServiceReplica.OnCloseAsync(cancellationToken);
 
+            ServiceTrace.Source.WriteInfoWithId(
+                TraceType,
+                this.traceId,
+                "Completed call to userServiceReplica.OnCloseAsync().");
+
             if (this.stateProviderReplica != null)
             {
+                ServiceTrace.Source.WriteInfoWithId(

```

Comment:
The indentation here and on 219 looks odd.

Reply:
Fixed.